src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -91,34 +140,38 @@ class VirtualMachine(
                     if (statement is ReturnNode || value !is EmptyValue) return value
 
                 }
+
                 fetchNextStatement()
             }
             return EmptyValue
         }
 
-        private fun executeStatement(statement: StatementNode): ExecValue =
-            when (statement) {
-                is ReturnNode -> executeExpression(statement.expression)
-                is FunctionNode -> {
-                    // just go onto next line, this is just a label
-                    EmptyValue
-                }
-                is SleepNode -> executeSleep(statement)
-                is AssignmentNode -> executeAssignment(statement)
-                is DeclarationNode -> executeAssignment(statement)
-                is MethodCallNode -> executeMethodCall(statement, false)
-                is FunctionCallNode -> executeFunctionCall(statement)
-                is IfStatementNode -> executeIfStatement(statement)
-                is StartStepIntoNode -> {
-                    stepInto = true
-                    EmptyValue
-                }
-                is StopStepIntoNode -> {
-                    stepInto = false
-                    EmptyValue
-                }
-                else -> EmptyValue
+        private fun getVariableState(): List<String>  {
+            return (0 until displayedDataMap.size).map { wrapString("${displayedDataMap[it]!!.first} = ${displayedDataMap[it]!!.second}") }
```

Comment:
`displayedDataMap.toSortedMap().values.map { wrapString("\"${it.first} = ${it.second}\"") }` might be a bit cleaner if you need the output in numerical order? Correct me if I'm missing something though